ddl/index.go

```diff
@@ -662,7 +662,7 @@ func (w *worker) onCreateIndex(d *ddlCtx, t *meta.Meta, job *model.Job, isPK boo
 		job.SnapshotVer = 0
 		job.SchemaState = model.StateWriteReorganization
 
-		if job.MultiSchemaInfo == nil {
+		if job.MultiSchemaInfo == nil && tblInfo.GetPartitionInfo() != nil {
```

Comment:
Why not let MultiSchemaChange go with distributed execution? @tangenta we need support with several adding indexes in one statement.

Reply:
I don't change the behavior. Which is the same as before.

Reply:
If we have considered it in the test plan, supporting multi-schema change is fine.

Reply:
Yes, it is better include all add index subjob could go distributed way.